src/state/pools/fetchPools.ts

```diff
@@ -51,26 +52,25 @@ export const fetchPoolsBlockLimits = async () => {
   })
 }
 
-export const fetchPoolsTotalStaking = async () => {
-  const nonBnbPools = poolsConfig.filter((p) => p.stakingToken.symbol !== 'BNB')
-  const bnbPool = poolsConfig.filter((p) => p.stakingToken.symbol === 'BNB')
-
-  const callsNonBnbPools = nonBnbPools.map((poolConfig) => {
-    return {
-      address: poolConfig.stakingToken.address,
-      name: 'balanceOf',
-      params: [getAddress(poolConfig.contractAddress)],
-    }
-  })
+const nonBnbPools = poolsConfig.filter((p) => p.stakingToken.symbol !== 'BNB')
```

Comment:
we can use lodash partition, something like this

`
 const [bnbPool, nonBnbPools] = partition(pools, (p) => p.stakingToken.symbol === 'BNB')
`

Reply:
@memoyil @chef-jojo we might also just ignore the split into 2 arrays and just use whole thing since later on:
1. bnb.address is actually configured as wbnb.address anyway
2. wbnbABI and cakeABI are the same ERC20 stuff, at least here we just need `balanceOf`. The ABI entry for it is slightly different but mostly just regarding name and other non-important stuff. 🤔 

We perhaps should use erc20.json here to begin with 🤷

Reply:
Good point! I will refactor it following this. Actually I just moved this out of runtime function 😅